pkg/cosign/cosign.go

```diff
@@ -408,7 +416,7 @@ func extractIssuer(imgRef string, payload []payload.SimpleContainerImage, log lo
 		if issuer := p.Optional["Issuer"]; issuer != nil {
 			return issuer.(string), nil
 		} else {
-			log.Info("failed to extract image issuer from verification response", "image", imgRef, "payload", p)
+			log.V(3).Info("failed to extract image issuer from verification response", "image", imgRef, "payload", p)
 			return "", fmt.Errorf("unknown image response for " + imgRef)
```

Comment:
why do we have a return statement in both if and else block of this [loop](https://github.com/kyverno/kyverno/pull/2973/files#diff-19f451f34e955fe33fac7e12f3099cc5eb3b05252fc467109736696b3e971265R415)? this way we are only checking the first element of the payload array